hw/top_earlgrey/data/chip_conn_testplan.hjson

```diff
@@ -0,0 +1,776 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: "chip_conn"
+
+  testpoints: [
+
+    /////////////////////////
+    // AON_TIMER resets    //
+    /////////////////////////
+    {
+      name: AON_TIMER_RST
+      desc: ''Verify rstmgr's resets_o is connected to aon_timer's reset port.''
+      milestone: V1
```

Comment:
Lets put milestone: V2 since we added most of these in V2 phase (connectivity is "plan B" when we are unable to verify it in a functional DV V2 test).

Reply:
Sounds good! Will change it.